src/pages/Login/Login.js

```diff
@@ -1,12 +1,114 @@
-import React from "react";
+import React, { useState } from 'react';
 import './Login.scss';
+import Logo from './logo.png';
+import { useNavigate } from 'react-router-dom';
 
 const Login = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const setChangeEmail = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const setChangePassword = (event) => {
+    setPassword(event.target.value);
+  };
+
+  const goToSignUp = () => {
+    navigate('./sign-up');
+  };
+
+  const goToMain = () => {
+    fetch('http://10.58.52.64:8000/users/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
```

Comment:
객체에서 `key값`과 `value로 사용되는 변수명`이 동일하면 이렇게 작성하시면 됩니다.
```suggestion
      body: JSON.stringify({
        email,
        password,
      }),
```